processor/src/test/java/org/mapstruct/ap/test/bugs/_913/Issue913SetterMapperForCollectionsTest.java

```diff
@@ -63,6 +63,8 @@
      * The null value mapping strategy on type level (Mapper) should generate forged methods for the
      * conversion from string to long that return null in the entire mapper, so also for the forged
      * mapper. Note the default NVMS is RETURN_NULL.
+     *
+     * The result of the forged methods should be applied directly to the target object.
```

Comment:
That is a good idea. I can make a separate PR only with the Javadoc changes of the test methods